idea-plugin/src/main/kotlin/arrow/meta/ide/plugins/proofs/intentions/implicitcoercion.kt

```diff
@@ -1,37 +1,43 @@
 package arrow.meta.ide.plugins.proofs.intentions
 
 import arrow.meta.ide.IdeMetaPlugin
+import arrow.meta.ide.plugins.proofs.intentions.PairTypes.Companion.pairOrNull
 import arrow.meta.phases.CompilerContext
 import arrow.meta.phases.ExtensionPhase
 import arrow.meta.plugins.proofs.phases.areTypesCoerced
+import com.intellij.codeInspection.ProblemHighlightType
 import org.jetbrains.kotlin.idea.debugger.sequence.psi.resolveType
 import org.jetbrains.kotlin.psi.KtDotQualifiedExpression
 import org.jetbrains.kotlin.psi.KtElement
-import org.jetbrains.kotlin.types.KotlinType
 
 fun IdeMetaPlugin.makeImplicitCoercionIntention(compilerContext: CompilerContext): ExtensionPhase =
-  addIntention(
-    text = "Make coercion implicit",
+  addApplicableInspection(
+    defaultFixText = "Make_coercion_implicit",
```

Comment:
this instance has to enabled by Default

Reply:
it is already because the parameter is true by default, but I'll add the explicit argument anyway 👍

Reply:
It didn't show up in my ide. But we already came across that issue. I'll deprecate the default behaivor in `enableByDefault` for that matter